cmd/root.go

```diff
@@ -3,10 +3,11 @@ package cmd
 import (
 	"os"
 
-	admissionv1 "k8s.io/api/admissionregistration/v1"
+	"github.com/jet/kube-webhook-certgen/pkg/k8s"
```

Comment:
this is the same repo which we forked from, we should merge this code if not exist inside.

Reply:
yes，I add the code by myself